test/passing/tests/wrap_comments.ml.ref

```diff
@@ -147,6 +159,6 @@ let _ =
 
 let _ =
   (*
-     blah blah
+    blah blah
```

Comment:
This is an improvement. The extra indentation was caused by a trailing space on the first line.